Views/Metas.liquid

```diff
@@ -0,0 +1,29 @@
+﻿{% SetMeta, Name: "viewport" , Content: "width=device-width, initial-scale=1.0" %}
+
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/favicon.ico", Rel: "shortcut icon", Type: "image/x-icon" %}
```

Comment:
Ezekben az URL-ekben már nincs benne egy kezdő tilde, ami biztosítja, hogy akkor is működjenek, ha egy URL prefix alatt van a tenant (tehát example.com/mytenant, nem mytenant.com). Működnek úgy is?

Reply:
Media Theme-ként deployolva mindenesetre nem fog működni, mert azok a fájlok nem a /Themes alatt vannak, hanem egy másik domain alatt. A Href taget kellene használni, ahogy a https://github.com/Lombiq/Orchard-Liquid-Markup#helpers alatt le is van írva.

Reply:
Oké, én használok Href taget, de akkor minden egyes képhez kell egy? Akkor nem lenne egyszerűbb elhagyni a RegisterLink-et és a SetMeta-t? A Try Orchard-os témában is úgy van, és az elég új még.

Reply:
Tényleg már lehet úgy túl bonyolult lesz, igen, igazad van, legyen úgy, ahogy a Try Orchardban van.